### ISSUE TITLE
ColorPicker - hue slider does not work if used with keyboard navigation

### ISSUE THREAD

**Issue by lidiaCirrone** [chakra-ui/zag#2389]
# 🐛 Bug report

The hue slider inside the ColorPicker dialog, if accessed by navigating with the keyboard, does not work properly: the thumb does not move and a "Uncaught Error: Unknown color channel: hue" is raised (specifically, NextJS 14, getChannelRange, incrementChannel, ArrowRight, onKeyDown")

## 💥 Steps to reproduce

1. Go to the ColorPicker page https://zagjs.com/components/react/color-picker
2. Press the Tab and Enter keys to reach and open the color picker modal, then tab again until you reach the hue slider
3. Press the arrow keys to move the thumb
4. The thumb does not move and an error is raised in the console

## 💻 Link to reproduction

https://zagjs.com/components/react/color-picker

## 🧐 Expected behavior

The thumb should move either left or right, depending on the key that's been pressed. If that's of any help, the slider right under it, the saturation one, works as expected

## 🌍 System information

Happens in both Mozilla Firefox and Google Chrome (latest versions)

## 📝 Additional information

https://github.com/user-attachments/assets/14725433-82b6-4735-94fa-156c4c0b4096

**Comment by segunadebayo** [comment]
This is a demo bug instead. Pushed a fix for it.

**Comment by lidiaCirrone** [comment]
Wow, that was super fast, thank you! Will it be fixed in Chakra UI, too?
